apps/docs/src/app/documentation/core-helpers/code-example/code-example.component.ts

```diff
@@ -131,7 +133,11 @@ export class CodeExampleComponent implements OnInit, AfterViewInit {
         this.exampleFiles.forEach(example => {
 
             if (example.fileName && example.component) {
-                this.parameters.html_tag = 'fd-' + example.fileName;
+
+                if (example.fileName !== undefined && example.fileName.indexOf('platform') !== -1) { this.parameters.html_tag = 'fdp-' + example.fileName; }
```

Comment:
Try to keep 
```
if () {
 
} else {

}
```
structure of statements.

Reply:
updated